core/constants_test.py

```diff
@@ -122,3 +122,8 @@ def test_language_constants_are_in_sync(self) -> None:
             set(rtl_audio_languages) & set(ltr_content_languages)
         )
         self.assertFalse(conflicts)
+
+    def test_constants_can_be_set(self) -> None:
+        """Test __setattr__ to see if constants can be set as needed."""
+        with self.swap(constants.constants, 'TESTING_CONSTANT', 'test_2'):
+            self.assertEqual(constants.constants.TESTING_CONSTANT, 'test_2')
```

Comment:
I'm bit confused, what line this actually covers?

Reply:
It covers Line 69 of core/constants.py. This is what happens when you follow the instructions listed on #14219 on the develop branch:

<img width="374" alt="Screen Shot 2022-04-27 at 11 36 26 AM" src="https://user-images.githubusercontent.com/40303727/165556572-0bb4d506-f9c4-45fb-a9f5-8d83aa41c1b5.png">

Reply:
Oh sorry realized I forgot to unassign myself. @vojtechjelinek PTAL